packages/keyboard-key/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@fluentui/keyboard-key",
+  "version": "0.1.0",
+  "description": "A simple utility for determining the KeyboardEvent.key property from a keyboard event.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
```

Comment:
This can be removed unless you add back the `@uifabric/set-version` dep (which I don't totally understand why we have that everywhere?)